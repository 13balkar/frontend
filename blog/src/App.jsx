import React from 'react';
import { BrowserRouter, Route, Routes } from 'react-router-dom';
import { ERROR_ROUTE, HOME_ROUTE } from './constants/routes';
import { BlogPostProvider } from './contexts/blogPostContext';
import Error from './pages/error';
import Homepage from './pages/homepage';
import PageNotFound from './pages/pageNotFound';
function App () {
  return (
    <BrowserRouter>
      <Routes>
        <Route path={HOME_ROUTE} element={
          <BlogPostProvider>
            <Homepage />
          </BlogPostProvider>} />
        <Route path={`${ERROR_ROUTE}/:errorCode?`} element={<Error />} />
        <Route path="*" element={<PageNotFound />} />
      </Routes>
    </BrowserRouter>
  );
}

export default App;
