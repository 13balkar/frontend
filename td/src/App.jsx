import React from 'react';
import { BrowserRouter, Route, Routes } from 'react-router-dom';
import Error from './pages/error';
import HomePage from './pages/homepage';
import LoginPage from './pages/login';
const App = () => {
  return (
    <BrowserRouter>
    <Routes>
    <Route path='/' element={<LoginPage />} />
    <Route path='/todo' element={<HomePage />} />
    <Route path='/error/:errorCode' element={<Error />} />
    </Routes>
    </BrowserRouter>
  );
};
export default App;
