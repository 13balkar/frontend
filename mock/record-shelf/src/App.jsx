import './App.css';
import React from 'react';
import { BrowserRouter, Route, Routes } from 'react-router-dom';
import { SyncPage, SongPage } from './pages';
import { HOME_PATH, SONGS_PATH } from './constants/routes';
const App = () => {
  return (
    <BrowserRouter>
      <Routes>
        <Route path={HOME_PATH} element={<SyncPage />} />
        <Route path={SONGS_PATH} element={<SongPage />}/>
      </Routes>
    </BrowserRouter>
  );
};

export default App;
