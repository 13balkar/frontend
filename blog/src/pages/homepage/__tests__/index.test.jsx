import { render } from '@testing-library/react';
import * as React from 'react';
import App from '..';

describe('Home Page', () => {
  it('should render correctly', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
