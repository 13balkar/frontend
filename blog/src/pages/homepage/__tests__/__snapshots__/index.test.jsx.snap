// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page should render correctly 1`] = `
<DocumentFragment>
  <div
    class="app"
  >
    <header
      class="outerPadding"
    >
      <div
        class="headerName"
      >
        <h2>
          The Artifact
        </h2>
        <h3>
          <i>
            Culture and Art Blog
          </i>
        </h3>
      </div>
      <nav
        class="headerNav"
      >
        <a
          href=""
        >
          Blog
        </a>
        <a
          href=""
        >
          About
        </a>
        <a
          href=""
        >
          Contact
        </a>
      </nav>
    </header>
    <h1>
      Loading
    </h1>
    <footer
      class="outerPadding"
    >
      <p>
        Â©artifact.com 2019
      </p>
    </footer>
  </div>
</DocumentFragment>
`;
